lambda/src/main/scala/pricemigrationengine/services/CohortTableLive.scala

```diff
@@ -175,9 +175,27 @@ object CohortTableLive {
                     _ => Logging.info(s"Wrote $result to Cohort table")
                   )
               }.provide(dependencies)
+
+              override def fetchAll(): IO[CohortFetchFailure, ZStream[Any, CohortFetchFailure, CohortItem]] = {
+                for {
+                  cohortTableConfig <- CohortTableConfiguration.cohortTableConfig
```

Comment:
Could `cohortTableConfig` be brought in during the impl construction like `stageConfig` at [line 117](https://github.com/guardian/price-migration-engine/blob/98cac5036bbf612c574b6bdae00a85799b99880f/lambda/src/main/scala/pricemigrationengine/services/CohortTableLive.scala#L117)?
Then it could give a `ConfigurationFailure` before trying to fetch.